src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -2,10 +2,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from flask import Flask, jsonify
```

Comment:
in folder src/promptflow/promptflow/_sdk/_service/, add generated swagger.json and steps to generate it in README.md